src/actions/graph.test.js

```diff
@@ -46,19 +46,19 @@ describe('graph actions', () => {
       });
     });
 
-    it('uses newgraph if the flag is set', () => {
-      const state = reducer(mockState.animals, changeFlag('newgraph', true));
+    it('uses new graph if the oldgraph flag is set to false', () => {
+      const state = reducer(mockState.animals, changeFlag('oldgraph', false));
       const store = createStore(reducer, state);
       return calculateGraph(getGraphInput(state))(store.dispatch).then(() => {
-        expect(store.getState().graph.newgraph).toBe(true);
+        expect(store.getState().graph.oldgraph).toBe(false);
       });
     });
 
     it('uses dagre if the flag is not set', () => {
```

Comment:
This test should be renamed, as it will use the newgraph algo (not dagre) if the flag is not set - which was the original point of the test: Checking the default state.

One of these two tests should probably use the default state, and they should both be renamed, e.g:
"uses new graph if the oldgraph flag is not set"
"uses dagre if the oldgraph flag is set to true"

Reply:
Thank you - I've updated the naming of the tests